client/blocks/signup-form/passwordless.jsx

```diff
@@ -279,8 +282,8 @@ class PasswordlessSignupForm extends Component {
 			);
 		}
 		const submitButtonText = isSubmitting
-			? this.props.submitButtonLoadingLabel || this.props.translate( 'Creating Your Account…' )
-			: this.props.submitButtonLabel || this.props.translate( 'Create your account' );
+			? this.props.submitButtonLoadingLabel || this.props.translate( 'Creating account.' )
+			: this.props.submitButtonLabel || this.props.translate( 'Create an account' );
```

Comment:
I think this may change the default submit button value for everyone using passwordless.

We could instead leverage `submitButtonText` prop in `invite-accept-logged-out` and pass it down to `SignupForm` and then `Passwordless`. Same thing for the loading label.

Reply:
Make sense, Thank you!